src/diffusers/pipelines/pyramid_broadcast_utils.py

```diff
@@ -166,51 +106,68 @@ def enable_pyramid_attention_broadcast(
                 different tolerances to how much attention computation can be reused based on the differences between
                 successive blocks. So, this parameter must be adjusted per model after performing experimentation.
                 Setting this value to `2` is recommended for different models PAB has been experimented with.
+            temporal_attn_skip_range (`int`, *optional*):
+                The attention block to execute after skipping intermediate temporal attention blocks. If set to the
+                value `N`, `N - 1` attention blocks are skipped and every N'th block is executed. Different models have
+                different tolerances to how much attention computation can be reused based on the differences between
+                successive blocks. So, this parameter must be adjusted per model after performing experimentation.
+                Setting this value to `4` is recommended for different models PAB has been experimented with.
```

Comment:
(doc nit): we could consider adding the additional piece of info as a "Tip". This way it would stand out better for a user.